apiserver/facades/client/controller/controller_test.go

```diff
@@ -357,15 +340,8 @@ func (s *controllerSuite) makeCloudSpec(c *tc.C, pSpec *params.CloudSpec) enviro
 func (s *controllerSuite) TestHostedModelConfigs_CanOpenEnviron(c *tc.C) {
 	defer s.setupMocks(c).Finish()
 	c.Skip("Hosted model config is skipped because the tests aren't wired up correctly")
```

Comment:
Can you move this file over to the new way we skip methods. Implement `TestStub`. The skip text here doesn't tell us enough about what should happen.

Reply:
Consistency

Reply:
Sure, I misunderstood you before. Done now!